### DISCUSSION TITLE
Controlling styles for .. versionadded:: .. versionchanged:: .. versionremoved:: .. deprecated::

### DISCUSSION THREAD

**Discussion by BobDenny** [brechtm/rinohtype#440] [Q&A]
These appear to be controlled by the TextStyle `[versionmodified]`. Starting with this rendering which uses `.. versionadded::`
![Snap2](https://github.com/user-attachments/assets/7e979d57-3727-407d-848c-a4964c429d16)
I'm looking for styling the "New in" paragraph  and for separating the summary line below. Here it is in html:
![Snap4](https://github.com/user-attachments/assets/4c1dcfca-d985-4bbe-a542-928fe0217ffb)
and here is the source:
![Snap3](https://github.com/user-attachments/assets/d9928fe0-d593-47d1-b554-a1bb95417850)
I have attached my stylesheet which overrides `sphinx.rts`, as well as an excerpt from the `.stylelog`. 
```
[versionmodified]
base=DEFAULT_STYLE
font_color = #6ab0de
font_slant = ITALIC
```
Although I can see in the style log that it picks up `[versionmodified]`
```
StaticGroupedFlowables()   D:103 (\dev\astro\ASCOM Master Docs 2022\source\telescope.rst)   <versionmodified>
```
Nothing is applied. Sphinx is converting the `.. versionadded::` to "New in version 3" OK, and that is appearing in the PDF. But it appears that the `[versionmodified]` style isn't being applied. I'm not seeing ITALIC or the color. Also in sphinx.rts it's styled 
```
[versionmodified]
font_slant=ITALIC
``` 
and that's not being picked up either. I hope I haven't missed something silly. I need the color changes and that separation between the `.. versionadded::` section and the following paragraph.  Ideally I'd like the color to be applied only to the "New in" text like it is in the HTML. Can someone help me? 

[Telescope.Action.Legend.stylelog.txt](https://github.com/user-attachments/files/17752479/Telescope.Action.Legend.stylelog.txt)
[ascom.rts.txt](https://github.com/user-attachments/files/17752586/ascom.rts.txt)
[ascom.rtt.txt](https://github.com/user-attachments/files/17752593/ascom.rtt.txt)

**Comment by brechtm** [comment]
Better support for versionmodified admonitions was added in d58bf18cc187e989dca5bc38a8b1e7ea67144ab8. If you [install the current development version](https://github.com/brechtm/rinohtype#getting-started), you can override their styles defined in the Sphinx stylesheet (which is the default).
```ini
[versionmodified : Admonition]
base=builtin admonition
space_above=2pt
space_below=2pt
padding_top=0
padding_bottom=0pt
border_top=none
border_bottom=none

[versionmodified inline title : StyledText]
base=admonition inline title
# title already includes a trailing space
after = ''

[versionadded]
base=versionmodified

[versionadded inline title]
base=versionmodified inline title

[versionchanged]
base=versionmodified

[versionchanged inline title]
base=versionmodified inline title

[versionremoved]
base=versionmodified

[versionremoved inline title]
base=versionmodified inline title

[deprecated]
base=versionmodified

[deprecated inline title]
base=versionmodified inline title
```

So, by default all admonitions use the same styling defined by _versionmodified_. The specific styles can override part of or all of the style definition.
